spec/system/planning_applications/assessing_spec.rb

```diff
@@ -154,4 +154,6 @@
   end
 
   include_examples "assessor decision error message"
+
+  include_examples "assessor refused decision without public_comment error message"
```

Comment:
Is this the only place where we're using this new shared example? If so I think it's best to remove the shared example and keep that example here in this spec. My reasoning is that with the single use shared example the reader has to jump to another file to read it, but we're not getting the benefit of reducing duplication across our specs.